### PR TITLE
Treat raw IRI as an OWLClass in logical template cells

### PR DESCRIPTION
**Pull request by jamesaoverton**
When ROBOT expects to see a class label or CURIE in a template cell, and it sees a raw IRI instead, interpret it as an OWLClass.

- [X] tests have been added/updated
- [X] `mvn verify` says all tests pass
- [ ] `CHANGELOG.md` has been updated

This is in response to a report on Slack.

### CODE REVIEW

File: robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java [hunk 1]
```diff
@@ -991,6 +991,14 @@ protected static OWLClassExpression tryParse(
         // If we have a checker, try to get the class by label
         // This allows class labels with single quotes
         expr = checker.getOWLClass(content, false);
+        if (expr == null) {
+          // If not found by label, is it just an IRI?
+          IRI iri = IRI.create(content);
```
Comment (balhoff) [inline robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java:996]: @jamesaoverton if someone enters a CURIE using an unknown prefix, is there a danger of turning it into an IRI here? In my experience OWL API will turn those into fake IRIs where the prefix becomes a scheme, e.g., "UNKNOWN:1234" -> `<UNKNOWN:1234>`. I find those mistakes really maddening (just speaking in general, not about robot template). Should you make some assumptions about IRIs here, such as only doing this if it starts with `http:`, `https:`, `urn:`, etc.?

**Review by balhoff — COMMENTED** [review]
Seems good but I had one question.

File: robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java (hunk 1 above)
Comment (jamesaoverton) [reply robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java:996]: Good point. I agree.

Comment (jamesaoverton) [reply robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java]: I added a unit test that 'UNKNOWN:1234' fails, and it did without any other changes. But I also added a check for HTTP IRIs.

**Pull request merged**